qt/python/mantidqt/mantidqt/widgets/workspacedisplay/table/presenter.py

```diff
@@ -133,15 +132,9 @@ def supports(cls, ws):
         return TableWorkspaceDisplayModel.supports(ws)
 
     def replace_workspace(self, workspace_name, workspace):
-        if self.presenter.model.workspace_equals(workspace_name):
-            # stops triggering itemChanged signal while the data is being reloaded
-            self.presenter.view.blockSignals(True)
```

Comment:
I think removing the `blockSignals` here might be the cause of the flickering? Could we put it back?

Reply:
I have put them back in, but it still flickers

Reply:
Ok. I think it's still a good thing to keep to avoid too much extra work for big tables.

Reply:
just pushed it with them back in. I removed them by accident (I had this as a context manager and the block signals were done as part of that)